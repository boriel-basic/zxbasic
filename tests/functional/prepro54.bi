' Conditional evaluation tests
#define NAME 0

#if NAME
fail 'this should not appear
#endif

