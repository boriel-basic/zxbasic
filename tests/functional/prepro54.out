#line 1 "prepro54.bi"



#line 7 "prepro54.bi"

